client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -125,12 +127,17 @@ class DomainRegistrationSuggestion extends React.Component {
 			suggestion,
 			translate,
 			pendingCheckSuggestion,
+			siteHasDomainCredit,
 		} = this.props;
 		const { domain_name: domain } = suggestion;
 		const isAdded = hasDomainInCart( cart, domain );
 
 		let buttonContent;
 		let buttonStyles = this.props.buttonStyles;
+		const buttonCopyTest =
+			abtest( 'domainSelectButtonCopy' ) === 'purchase'
+				? translate( 'Purchase', { context: 'Domain mapping suggestion button' } )
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Purchase', { context: 'purchasing a theme'} )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/domain-select-button-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
I think we can switch to "Purchase" with a simple context "verb" - we already have that translated, and I think we don't need a seperate context for each "Purchase" button (other that verb vs/noun context).